worker/src/RTC/SimulcastConsumer.cpp

```diff
@@ -754,9 +754,13 @@ namespace RTC
 				// https://en.wikipedia.org/wiki/Audio-to-video_synchronization#Recommendations
 				static const uint32_t MaxExtraOffsetMs{ 75u };
 
+				// Outgoing packet matches the highest timestamp seen in the previous stream.
+				// Apply an expected offset for a new frame in a 30fps stream.
+				static const uint8_t MsOffset{ 33u }; // (1 / 30 * 1000).
+
 				int64_t maxTsExtraOffset = MaxExtraOffsetMs * this->rtpStream->GetClockRate() / 1000;
-				uint32_t tsExtraOffset =
-				  this->rtpStream->GetMaxPacketTs() - packet->GetTimestamp() + tsOffset;
+				uint32_t tsExtraOffset   = this->rtpStream->GetMaxPacketTs() - packet->GetTimestamp() +
+				                         tsOffset + MsOffset * this->rtpStream->GetClockRate() / 1000;
 
 				// NOTE: Don't ask for a key frame if already done.
```

Comment:
Shouldn't we move this condition above so we don't do unused computation?

Also, "make format" required in worker/ folder to fix lint issues.

Reply:
Sorry, which condition do u mean?

Reply:
Forget it, bad review from mobile.